lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -1035,6 +1035,10 @@ def install_repo_configs_from_url(host, repo_config_url, copy_dir = nil)
           end
           scp_to( host, repo, to_path )
 
+          if host['platform'] =~ /ubuntu-18.04/
```

Comment:
This appears to make the previous change redundant. The puppet-agent install helper, after setting this, immediately calls code that calls install_repo_configs_from_url.

Reply:
Tying it to apt-get update makes sense. We'll add a repo without signing here, the only other apt-get update corresponds to public repos which should be signed.

Reply:
The most relevant place might be in install_puppetlabs_dev_repo, since that's the specific type of repo we expect to be unsigned.